tests/integration/build.ps1

```diff
@@ -50,6 +50,7 @@ Function GetCakeCoreCLRPath([string]$ScriptRoot)
 # PREPARATION
 #####################################################################
 
+$Env:CAKE_INTEGRATION_TEST_ROOT = "../.."
```

Comment:
this should be after 
https://github.com/Kemyke/cake/blob/d8dd72d4ce3bdd6cd68bcf9c4db0655ff508b5e6/tests/integration/build.ps1#L174

where integration test variables are set,

Reply:
Also we would need to do the same in `build.sh`